### PR TITLE
Athlete dash feature dv

### PR DESCRIPTION
**Pull request by thedanielvogelsang**

### CODE REVIEW

File: app/controllers/players_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class PlayersController < ApplicationController
+  def index
+  end
+
+  def show
```
Comment (mdevoe12) [inline app/controllers/players_controller.rb:5]: @thedanielvogelsang 

Hey Dan,

Quick question on structure, is this the show page for the player's dashboard or for the player's profile?

Thanks!

Comment (thedanielvogelsang) [reply app/controllers/players_controller.rb:5]: I have been up until now considering them the same things. A player logs in and sees a (modified) version of the same show page shown to recruiters/the public. What are you thinking?

Comment (jimszalew) [reply app/controllers/players_controller.rb:5]: The way I see it, dashboard and profile seem like they would have the same  functions. But on the flip side of that, I feel like dashboard should be a user's first page when they log in. When logging into git hub you're not on your profile, you're on a dashboard. I guess to sum up my thoughts (using athlete as the example) athlete dashboard should be athlete_user facing, and athlete_profile should be recruiter/coach/other athletes facing.

Comment (mdevoe12) [reply app/controllers/players_controller.rb:5]: I feel similar to what Jim said regarding splitting the two apart. However, I'm not opposed to keeping Dashboard and Profile the same but rendering different views depending on who's logged in.

Dan, with your custom JS, does it have to be hardcoded to the dashboard, or can it be moved to a different view should it need to? I'm not sold either way, just wondering what the flexibility looks like.

**Pull request merged**